modules/page/html/render/public.js

```diff
@@ -135,7 +135,11 @@ exports.create = function (api) {
         h('button -pub -full', {
           'ev-click': api.invite.sheet
         }, '+ Join Pub'),
-        when(computed(channels, x => x.length), h('h2', 'Active Channels')),
+        when(computed(channels, x => x.length),
+             h('h2',
```

Comment:
It's a little bit strange having this as a hyperlink on the title as this is not a common pattern anywhere else in patchwork.

I would prefer if this were a seperate button at the bottom of the channel list "More Channels..."